src/theme.js

```diff
@@ -0,0 +1,29 @@
+export const theme = {
```

Comment:
Read this and try to use some default values for all you components. https://chakra-ui.com/docs/styled-system/theming/theme?scroll=true